packages/opam-monorepo/opam-monorepo.0.3.0/opam

```diff
@@ -0,0 +1,54 @@
+opam-version: "2.0"
+synopsis: "Assemble and manage fully vendored Dune repositories"
+description: """
+The opam monorepo plugin provides a convenient interface to bridge the
+opam package manager with having a local copy of all the source
+code required to build a project using the dune build tool."""
+maintainer: ["anil@recoil.org"]
+authors: [
+  "Anil Madhavapeddy" "Nathan Rebours" "Lucas Pluvinage" "Jules Aguillon"
+]
+license: "ISC"
+homepage: "https://github.com/ocamllabs/opam-monorepo"
+doc: "https://ocamllabs.github.io/opam-monorepo"
+bug-reports: "https://github.com/ocamllabs/opam-monorepo/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "ocaml" {>= "4.08.0"}
```

Comment:
```suggestion
  "ocaml" {>= "4.13"}
```
```
#=== ERROR while compiling opam-monorepo.0.3.0 ================================#
# context              2.1.2 | linux/x86_64 | ocaml-base-compiler.4.12.1 | pinned(https://github.com/ocamllabs/opam-monorepo/releases/download/0.3.0/opam-monorepo-0.3.0.tbz)
# path                 ~/.opam/4.12/.opam-switch/build/opam-monorepo.0.3.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p opam-monorepo -j 31 @install
# exit-code            1
# env-file             ~/.opam/log/opam-monorepo-1976-b81551.env
# output-file          ~/.opam/log/opam-monorepo-1976-b81551.out
### output ###
# (cd duniverse/0install/src/gui_gtk && /home/opam/.opam/4.12/bin/ocaml -I +compiler-libs /home/opam/.opam/4.12/.opam-switch/build/opam-monorepo.0.3.0/_build/.dune/default/duniverse/0install/src/gui_gtk/dune.ml)
# sh: 1: ocamlfind: not found
# (will skip building the GTK plugin)
# (cd _build/default && /home/opam/.opam/4.12/bin/ocamldep.opt -modules -impl cli/lock.ml) > _build/default/cli/.duniverse_cli.objs/lock.ml.d
# File "cli/lock.ml", line 259, characters 15-17:
# 259 |       let* acc in
#                      ^^
# Error: Syntax error: operator expected.
```

Reply:
Argh that's ocamlformat reformatting it this way. I'm afraid we'll have to abort this release and start again!

Reply:
Is there an issue somewhere tracking this bug?

Reply:
It is not an ocamlformat-per-se bug. It just defaults to format for the currently active OCaml version (I assume) and we didn't tell it to stay 4.08 compatible: https://github.com/ocamllabs/opam-monorepo/pull/287

Reply:
ah I see. I was unaware of the `ocaml-version` option

Reply:
I’m still considering this somewhat of a bug personally and I opened a PR here to avoid this issue in the future: https://github.com/ocaml-ppx/ocamlformat/pull/2064